spec/models/email_processor_spec.rb

```diff
@@ -94,17 +94,37 @@
 
     context "when the entry is a response to yesterday's email" do
       it "sets the entry date to yesterday's date" do
```

Comment:
Sounds good to me. I just cherry-picked your https://github.com/codecation/trailmix/commit/6c47464863a742e55da57807453d64e41e707592